src/stories/Spacing.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+type SizeProps = {
+  [key: string]: {
+    [key: string]: string;
+  };
+};
+
+export const SpacingBlocks = () => {
+  const sizes: SizeProps = {
+    xl: {
+      color: '--color-green-50',
+      mqBase: '32',
+      mqSmall: '48',
+      mqMedium: '64'
+    },
+    lg: {
+      color: '--color-blue-50',
+      mqBase: '16',
+      mqSmall: '24',
+      mqMedium: '32'
+    },
+    md: {
+      color: '--color-orange-50',
+      mqBase: '8',
+      mqSmall: '12',
+      mqMedium: '16'
+    },
+    sm: {
+      color: '--color-red-50',
+      mqBase: '6',
+      mqSmall: '8',
+      mqMedium: '8'
+    },
+    xs: {
+      color: '--color-amber-40',
+      mqBase: '4',
+      mqSmall: '4',
+      mqMedium: '4'
+    }
+  };
+
+  return (
+    <table>
+      <caption>Standard space units and their sizes across breakpoints</caption>
+      <thead>
+        <tr>
+          <th scope="col">Space unit</th>
+          <th scope="col">Default size, small screens up to 767 pixels</th>
+          <th scope="col">Medium screens from 768 up to 1023 pixels</th>
+          <th scope="col">Large screens 1024 pixels wide and above</th>
+        </tr>
+      </thead>
+      <tbody>
+        {Object.entries(sizes).map(([key, value]) => {
+          const size = `--space-${key}`;
+          return (
+            <tr key={size}>
+              <td>
+                <figure>
+                  <div
+                    className="sb-space"
+                    style={{
+                      backgroundColor: `var(${value.color})`,
+                      height: `var(${size})`,
+                      width: `var(${size})`
+                    }}
+                  />
+                  <figcaption className="sb-swatch__name">
+                    <dl>
+                      <dt className="u-visually-hidden">Space unit name</dt>
+                      <dd>{key.toUpperCase()}</dd>
+                      <dt className="u-visually-hidden">CSS variable</dt>
+                      <dd className="sb-variable">{size}</dd>
+                    </dl>
+                  </figcaption>
+                </figure>
+              </td>
+              <td>{value.mqBase} pixels</td>
+              <td>{value.mqSmall} pixels</td>
+              <td>{value.mqMedium} pixels</td>
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+  );
+};
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
Is it possible to add `export default SpacingBlocks` to avoid having to disable eslint?

Reply:
It seems I can remove the disable eslint lines :) - it must have been causing an issue at some point but no longer it seems.

As an aside, Storybook uses the [default export](https://storybook.js.org/docs/react/writing-stories/introduction#default-export) in a very specific way. Using [Button](https://github.com/wellcometrust/design-system/blob/develop/src/components/Button/Button.stories.tsx) as an example, the default export is effectively a config. If it is omitted the `.tsx` file is not used to create a storybook page.a

MDX files are slightly different. They are much more ideal for narrative style documentation, hence being used for the global settings. `Spacing.stories.mdx` is automatically added as a Storybook page and the companion `Spacing.stories.tsx` is used to provide the JSX elements.

Colours, Icons and Typography also work this way.

Reply:
Ace, thanks - makes sense. Might be worth checking why eslint is no longer flagging an issue here, though, as it could be covering a bigger issue?

Reply:
Good call 👍 . ES lint appears to be working fine. I put it down to some kind of dependency shenanigans (I've had lots of that recently) but will keep an eye on it.